exercises/dnd-character/runtests.jl

```diff
@@ -0,0 +1,64 @@
+# canonical data version: 1.1.0
+using Test
+
+include("dnd-character.jl")
+
+function ischaracter(dnd)
```

Comment:
The parameter is a character, so I think `c` or `character` instead of `dnd` would be a better name.